src/darwin/Framework/CHIP/templates/tests/CHIPClustersTests.zapt

```diff
@@ -1,237 +0,0 @@
-}
-
-- (void)onCommissioningComplete:(NSError *)error
-{
-    XCTAssertEqual(error.code, 0);
-    [_expectation fulfill];
-    _expectation = nil;
-}
-
-- (void)onAddressUpdated:(NSError *)error
-{
-    XCTAssertEqual(error.code, 0);
-    [_expectation fulfill];
-    _expectation = nil;
-}
-@end
-
-@interface CHIPClustersTests : XCTestCase
-@end
-
-@implementation CHIPClustersTests
-
-- (void)setUp {
-    [super setUp];
-    [self setContinueAfterFailure:NO];
-}
-
-- (void)testInitStack
-{
-    XCTestExpectation * expectation = [self expectationWithDescription:@"Pairing Complete"];
-
-    __auto_type * factory = [MatterControllerFactory sharedInstance];
-    XCTAssertNotNil(factory);
-
-    __auto_type * storage = [[CHIPTestStorage alloc] init];
-    __auto_type * factoryParams = [[MatterControllerFactoryParams alloc] initWithStorage:storage];
-    factoryParams.port = @(kLocalPort);
-
-    BOOL ok = [factory startup:factoryParams];
-    XCTAssertTrue(ok);
-
-    __auto_type * params = [[CHIPDeviceControllerStartupParams alloc] initWithKeypair:nil];
-    params.vendorId = kTestVendorId;
-    params.fabricId = 1;
-
-    // TODO: Once we have a non-nil keypair, use startControllerOnNewFabric.
-    CHIPDeviceController * controller = [factory startControllerOnExistingFabric:params];
-    XCTAssertNotNil(controller);
-
-    sController = controller;
-
-    CHIPToolPairingDelegate * pairing = [[CHIPToolPairingDelegate alloc] initWithExpectation:expectation];
-    dispatch_queue_t callbackQueue = dispatch_queue_create("com.chip.pairing", DISPATCH_QUEUE_SERIAL);
-
-    [controller setPairingDelegate:pairing queue:callbackQueue];
-
-    NSError * error;
-    [controller pairDevice:nodeId
-                   address:kAddress
-                      port:kRemotePort
-             discriminator:kDiscriminator
-              setupPINCode:kSetupPINCode
-                     error:&error];
-    XCTAssertEqual(error.code, 0);
-
-    [self waitForExpectationsWithTimeout:kPairingTimeoutInSeconds handler:nil];
-
-    __block XCTestExpectation * connectionExpectation = [self expectationWithDescription:@"CASE established"];
-    [controller getConnectedDevice:nodeId
-                             queue:dispatch_get_main_queue()
-                 completionHandler:^(CHIPDevice * _Nullable device, NSError * _Nullable error) {
-                     XCTAssertEqual(error.code, 0);
-                     [connectionExpectation fulfill];
-                     connectionExpectation = nil;
-                 }];
-    [self waitForExpectationsWithTimeout:kCASESetupTimeoutInSeconds handler:nil];
-}
-
-- (void)testShutdownStack
-{
-    CHIPDeviceController * controller = sController;
-    XCTAssertNotNil(controller);
-
-    [controller shutdown];
-    XCTAssertFalse([controller isRunning]);
-
-    [[MatterControllerFactory sharedInstance] shutdown];
-    XCTAssertFalse([[MatterControllerFactory sharedInstance] isRunning]);
-}
-
-- (void)testReuseChipClusterObject
```

Comment:
Don't we still want this test?  That is, don't we want to remove the YAML bits but keep this API test?